conan/api/output.py

```diff
@@ -85,13 +85,31 @@ def _color_enabled(stream):
 
 
 class ConanOutput:
```

Comment:
The only question here that remains is: Should `write` and `writeln` be part of the API? I doubted it so I didn't document them, but hapoy to do so if we think it's necessary